cylc/flow/workflow_files.py

```diff
@@ -1698,6 +1698,24 @@ def get_run_dir_info(
     return relink, run_num, rundir
 
 
+def detect_both_flow_and_suite(path):
+    """Detects if both suite.rc and flow.cylc are in directory.
+    Raises:
+        WorkflowFilesError: If both flow.cylc and suite.rc are in directory
+    """
+    msg = (f"Both {WorkflowFiles.FLOW_FILE} and {WorkflowFiles.SUITE_RC} "
```

Comment:
IMO a link to https://cylc.github.io/cylc-doc/latest/html/7-to-8/summary.html#backward-compatibility would be helpful.
I've asked Element whether we can use a link shortener.

Reply:
Happy to add a link. I shall be making a doc PR to make this change clear to users too.
I personally would prefer not to use link shortener. I think getting the cylc url in there is good, sometimes I think link shorteners look a bit spammy, as always happy to be overruled though!